### ISSUE TITLE
broken android builds because of jcenter

### ISSUE THREAD

**Issue by maxkomarychev** [AppsFlyerSDK/appsflyer-react-native-plugin#348]
# Report

Android builds are failing because of jcenter

## Plugin Version

ℹ Please replace these two lines with the plugin version.

=> Found "react-native-appsflyer@6.4.0"

## On what Platform are you having the issue?

ℹ Please replace these two lines with the SDK version.
e.g. ios and android

android

## What did you do?

- tried to run android locally `react-native run-android`
- compile app in release mode in CI

## What did you expect to happen?

stuff to work

## What happened instead?

```
A problem occurred configuring project ':react-native-appsflyer'.
> Could not resolve all artifacts for configuration ':react-native-appsflyer:classpath'.
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72.
     Required by:
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:aaptcompiler:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.lint:lint-gradle-api:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:gradle-api:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > androidx.databinding:databinding-compiler-common:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools:sdk-common:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools:common:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools.build:manifest-merger:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools.analytics-library:tracker:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools:sdklib:27.0.0 > com.android.tools:repository:27.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > com.android.tools:sdk-common:27.0.0 > com.android.tools.analytics-library:shared:27.0.0
      > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72.
         > Could not get resource 'https://dl.bintray.com/android/android-tools/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.72/kotlin-stdlib-jdk8-1.3.72.pom'.
            > Could not HEAD 'https://dl.bintray.com/android/android-tools/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.72/kotlin-stdlib-jdk8-1.3.72.pom'. Received status code 502 from server: Bad Gateway
   > Could not resolve org.ow2.asm:asm:7.0.
     Required by:
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > org.ow2.asm:asm-commons:7.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > org.ow2.asm:asm-util:7.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build:builder:4.0.0 > org.ow2.asm:asm-tree:7.0
      > Could not resolve org.ow2.asm:asm:7.0.
         > Could not get resource 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm/7.0/asm-7.0.pom'.
            > Could not HEAD 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm/7.0/asm-7.0.pom'. Received status code 502 from server: Bad Gateway
   > Could not resolve org.ow2.asm:asm-analysis:7.0.
     Required by:
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > org.ow2.asm:asm-commons:7.0
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > org.ow2.asm:asm-util:7.0
      > Could not resolve org.ow2.asm:asm-analysis:7.0.
         > Could not get resource 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm-analysis/7.0/asm-analysis-7.0.pom'.
            > Could not HEAD 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm-analysis/7.0/asm-analysis-7.0.pom'. Received status code 502 from server: Bad Gateway
   > Could not resolve org.ow2.asm:asm:7.0.
     Required by:
         project :react-native-appsflyer > com.android.tools.build:gradle:4.0.0 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
      > Could not resolve org.ow2.asm:asm:7.0.
         > Could not get resource 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm/7.0/asm-7.0.pom'.
            > Could not HEAD 'https://dl.bintray.com/android/android-tools/org/ow2/asm/asm/7.0/asm-7.0.pom'. Received status code 502 from server: Bad Gateway
```

## Please provide any other relevant information.

bintray has been sunset and can not serve packages anymore

https://status.bintray.com/

**Comment by amit-kremer93** [comment]
@maxkomarychev we updated our build.gradle file and we will release new version soon.

**Comment by amit-kremer93** [comment]
issue is fixed in the latest release [6.4.40](https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/master/CHANGELOG.md#6440)
